src/report/dot/index.mjs

```diff
@@ -151,7 +151,7 @@ function pryReporterOptionsFromResults(pGranularity, pResults) {
   const lFallbackReporterOptions =
     pResults?.summary?.optionsUsed?.reporterOptions?.dot;
 
-  // using lodash.get here because the reporter options will contain nested
+  // using _.get here because the reporter options will contain nested
   // properties, which it handles for us
```

Comment:
```suggestion
  // using a bespoke 'get' function here because the reporter options will
  // contain nested properties, which it handles for us
```